scss/styles.scss

```diff
@@ -86,6 +86,8 @@
   height: 90px;
   justify-content: space-between;
   left: 0;
+  padding-left: 25px;
```

Comment:
use `padding: 0 25px;` instead. this is because it's easier to change, you edit only one property